packages/builtin/string.pony

```diff
@@ -978,16 +982,13 @@ actor Main
         while i < _size do
           (let c, let len) = utf32(i.isize())
 
-          if chars.contains(c) then
-            // If we find a delimeter, add the current string to the array.
-            occur = occur + 1
-
-            if (n > 0) and (occur >= n) then
-              break
-            end
-
-            result.push(cur = recover String end)
+          let found = match chars
```

Comment:
I can fix that.

Would be need an RFC to change the signature of the split method to work on `String val` instead?